ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1125,7 +1125,7 @@ static PrjFS_Result HandleFileNotification(
     // We have to convert to full for PreDelete until we have WILL_RENAME support.  The PreDelete could be for a rename.
```

Comment:
This comment should be updated to explain why we convert to full for `PreDeleteFromRename` (or it should be removed)